src/geojson/geojson.ml

```diff
@@ -319,6 +325,14 @@ module Make (J : Intf.Json) = struct
          ]
         @ bbox_to_json_or_empty bbox)
 
+    let to_json ?foreign_members t =
```

Comment:
This should be folded into the `to_json` function immediately above this one so that we have one `to_json` function:

```ocaml
let to_json ?bbox ?foreign_members (t, props) =
```